circleci.sh

```diff
@@ -137,8 +137,8 @@ fi
 docker run -it --rm \
        -e DOCKER_API_VERSION=${DOCKER_API_VERSION:-1.23} \
        -v /var/run/docker.sock:/var/run/docker.sock \
-       -v $(pwd):$(pwd) \
+       -v "$(pwd)":"$(pwd)" \
```

Comment:
This is usually written as `"$(pwd):$(pwd)"` (which is equivalent), but this is ok too.